### PR TITLE
Feature/extended event info

### PR DESCRIPTION
**Pull request by Nico-J**
This PR adds all relevant event information (such as event type, attendee email, cancel/reschedule links, ...) to the description of the event added to the integrations.

### CODE REVIEW

File: lib/emails/EventOrganizerMail.ts [hunk 1]
```diff
@@ -18,14 +20,24 @@ export default class EventOrganizerMail extends EventMail {
...
-      description: this.calEvent.description + this.stripHtml(this.getAdditionalBody()) + this.stripHtml(this.getAdditionalFooter()),
-      duration: { minutes: dayjs(this.calEvent.endTime).diff(dayjs(this.calEvent.startTime), 'minute') },
+      description:
+        this.calEvent.description +
+        stripHtml(this.getAdditionalBody()) +
+        stripHtml(this.getAdditionalFooter()),
+      duration: { minutes: dayjs(this.calEvent.endTime).diff(dayjs(this.calEvent.startTime), "minute") },
       organizer: { name: this.calEvent.organizer.name, email: this.calEvent.organizer.email },
-      attendees: this.calEvent.attendees.map( (attendee: any) => ({ name: attendee.name, email: attendee.email }) ),
+      attendees: this.calEvent.attendees.map((attendee: any) => ({
```
Comment (baileypumfleet) [inline lib/emails/EventOrganizerMail.ts:37]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689303)
```suggestion
      attendees: this.calEvent.attendees.map((attendee: unknown) => ({
```

Comment (baileypumfleet) [inline lib/emails/EventOrganizerMail.ts:37]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689303)
```suggestion
      attendees: this.calEvent.attendees.map((attendee: unknown) => ({
```

**Review by Malte-D — APPROVED** [review]
I like the new eventParser. The rest of the code is also working

**Comment by baileypumfleet** [comment]
If we can get the conflicts resolved, this can be merged in :)

File: lib/emails/EventOrganizerMail.ts (hunk 1 above)
Comment (Nico-J) [reply lib/emails/EventOrganizerMail.ts]: Fixed. :) Thanks for the hint.

**Pull request merged**